docs/source/install.rst

```diff
@@ -12,11 +12,11 @@ of this document.
 NCI Users
 =========
 
-Payu is available for all users on Raijin.
-
-To load payu, load the environment module::
-
-   module load payu
+Payu is available for all users on Gadi.
+But it needs to access the 'hh5' data in NCI at first, we could use that conda module to work payu functions.
```

Comment:
Users need to be a member of the `hh5` project. It is probably best to just point to the [instructions on the ACCESS-Hive](https://access-hive.org.au/models/run-a-model/run-access-esm/#model-specific-prerequisites), as they are being actively updated and supported.

Reply:
Yes, thanks for your remind of this instructions.